src/python_framework_v02/troute/nhd_io.py

```diff
@@ -322,27 +320,23 @@ def get_usgs_from_time_slices_csv(
         usgs_df.insert(i + 11, temp_name8, np.nan)
 
     usgs_df = usgs_df.interpolate(method="linear", axis=1)
-    usgs_df.drop(usgs_df[usgs_df.iloc[:,0] == -999999.000000].index , inplace=True)
+    usgs_df.drop(usgs_df[usgs_df.iloc[:, 0] == -999999.000000].index, inplace=True)
```

Comment:
If we are computing a mask, a call to drop is unnecessary
Assuming that first column of usgs_df is labeled "ColumnA"
```
usgs_df = usgs_df.loc[usgs_df["ColumnA"] == -999999]
```